autrainer/datasets/utils/dataloader.py

```diff
@@ -0,0 +1,24 @@
+from collections.abc import Mapping
+from typing import List
+
+import numpy as np
+import torch
+
+from .data_struct import Data
+
+
+def default_data_collator(features: List[Data]) -> Data:
```

Comment:
I think a docstring would be great here especially covering what happens with tensors, numpy arrays, or e.g. ints.